src/h5web/dataset-visualizer/vis/matrix/utils.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AccessorFunc = (row: number, col: number) => any;
+
+export interface CellSize {
+  width: number;
+  height: number;
+}
+
+export interface GridSettings {
+  cellSize: CellSize;
+  rowCount: number;
+  columnCount: number;
+}
```

Comment:
I put additional types in a `util.ts` but I wonder if this is worthwhile as there are few.

Reply:
`util.ts` removed